portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,31 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Cycles through images automatically
+ */
+function slideShow(ind = 0) {
```

Comment:
use more descriptive variable names (https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers) -- example: imageIndex